### ISSUE TITLE
Vérifier fonctionnement de 3 dessins

### ISSUE THREAD

**Issue by mdartic** [makinacorpus/accessimap-lecteur-der#40]

**Comment by mdartic** [comment]
[Carte_heuristique(1).zip](https://github.com/makinacorpus/accessimap-lecteur-der/files/760317/Carte_heuristique.1.zip)
[Quiz.francais(1).zip](https://github.com/makinacorpus/accessimap-lecteur-der/files/760318/Quiz.francais.1.zip)
[Lycee_Marmande(1).zip](https://github.com/makinacorpus/accessimap-lecteur-der/files/760319/Lycee_Marmande.1.zip)

**Comment by BastienAlvez** [comment]
Lycee_Marmande(1).zip ne comporte aucun data-link, il est donc impossible de retrouver les elements correspondant sur le lecteur, néanmoins il comporte la bonne propriété id, il serait donc simple de le formater correctement.

Carte_heuristique(1).zip et Quiz.francais(1).zip no comportent ni data-link, ni les bon id correspondant aux interactions.

**Comment by mdartic** [comment]
@GregIJA, peux tu nous dire si les dessins carte heuristique et quiz sont bien des dessins fait à la main, et qui ne respectent pas la 'méthode' de l'éditeur ?

**Comment by GregIJA** [comment]
Je regarde ça au plus vite.
De toute façon aucune de nos cartes n'a été intégralement faite avec l'éditeur.
Aura t-on un moyen simple de les migrer pour qu'elle soit lisible par le lecteur?

**Comment by mdartic** [comment]
Pour des dessins de type Lycee_Marmande, je pense que cela peut être fait à la main facilement.

Pour les carte_heuristique & quiz.français, je pense que ça va être beaucoup plus long.

Car le fichier interaction ne référence pas l'élément svg, mais une zone délimitée par une bounding box...

**Comment by GregIJA** [comment]
Je m'en occuperai à l'occasion si tu m'expliques tout ça.

**Comment by mdartic** [comment]
Ok, à rediscuter précisément.

Je te donne juste quelques éléments ici :
- le fichier `interactions.xml` créé par l'éditeur comporte des `actions`, corrélées à un `poi`, qui représente un ou plusieurs éléments du dessin
- la corrélation du `poi` avec les éléments du dessin s'effectue sur un identifiant (attribut `id` de l'élément xml `poi`) renseigné dans l'attribut `data-link` du ou des éléments du dessin
- par exemple, pour un poi avec l'`id` 'title', le lecteur cherche dans le dessin le ou les éléments qui ont l'attribut `data-link` avec la valeur 'title' 

Le dessin Lycee_Marmande n'a pas d'attributs `data-link` mais il a les attributs `id`.
Il est facile de rajouter les attributs `data-link` identiques aux `id`.

Les autres dessins n'ont aucune information `id` ou `data-link` sur les éléments.
Il sera donc plus compliqué de retrouver les correspondances entre les `poi` du fichier `interactions.xml` et les éléments du dessin.

Mieux ? A rediscuter si tu veux à  l'oral.

**Comment by GregIJA** [comment]
Je laisse ouvert pour le moment, je ferai des tests plus tard quand les filtres fonctionneront correctement.

**Comment by MedHediAmri** [comment]
J'ai fait les tests. Les interactions ne fonctionnent pas.

**Comment by mdartic** [comment]
Oui, il est précisé plus haut que les interactions ne sont pas au bon format.
Je ne pense pas que Grég ait eu le temps de corriger les fichiers.

**Comment by mdartic** [comment]
@GregIJA comment veux tu qu'on procède pour cette issue ?
Sachant que les DER ne sont pas au bon format ?

J'ai aussi regardé le DERi du Festival de rue
[Festival de rue.zip](https://github.com/makinacorpus/accessimap-lecteur-der/files/1421655/Festival.de.rue.zip)

Il n'est pas conforme à ce que l'éditeur produit et ce que le lecteur attend...

Souhaites tu qu'on les convertisse pour qu'ils deviennent valident ? (travail à la mano)
Ou tu t'en occupes ?
Ou on les recrée avec l'éditeur pour voir si on arrive à bien refaire un DERi équivalent ?
